pulsar-broker/src/main/java/org/apache/pulsar/broker/service/PulsarStats.java

```diff
@@ -239,6 +239,14 @@ public void recordTopicLoadTimeValue(String topic, long topicLoadLatencyMs) {
         }
     }
 
+    public void recordTopicLoadFailed(String topic) {
+        try {
+            brokerOperabilityMetrics.recordTopicLoadFailed();
+        } catch (Exception ex) {
```

Comment:
why catch exception?

Reply:
just following the old pattern: https://github.com/apache/pulsar/blob/master/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/PulsarStats.java#L234

Reply:
I don't think it needs catch exception, because we don't need to catch any RuntimeException here.